app/services/code_hosting/git_hub_service.rb

```diff
@@ -9,8 +10,10 @@ module FastlaneCI
   # Data source that interacts with GitHub
   class GitHubService < CodeHostingService
     include FastlaneCI::Logging
+    include FastlaneCI::GitHubHandler
 
     class << self
+      include FastlaneCI::GitHubHandler
```

Comment:
@snatchev I included this at both instance-level and class-level, is there a better way to accomplish this instead of doing 2 includes?

Reply:
@taquitos see here https://github.com/fastlane/ci/pull/612